src/sass/login.scss

```diff
@@ -1,5 +1,190 @@
 @import "shared-variables";
 
+.login {
+  background: $color-white;
+
+  hr {
+    border-color: $color-gray-lighter;
+  }
+
+  @include media-maxwidth($small-screen) {
+    .hide-for-small {
+      display: none;
+    }
+
+    .explanation-content {
+      padding: 1em 2em;
```

Comment:
I think you can remove this - I see you put it in to make the text break nicely, but it pads the text too much on  mobile. This content is subject to change any way so I think we don't need to fuss over it too much.